<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>567</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Signals Veiwer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>wicon.jpg</normaloff>wicon.jpg</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LeftButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Multi-Channel-Signal-Viewer-master/left.png</normaloff>../Multi-Channel-Signal-Viewer-master/left.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="RightButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Multi-Channel-Signal-Viewer-master/right.png</normaloff>../Multi-Channel-Signal-Viewer-master/right.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="UpButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Multi-Channel-Signal-Viewer-master/up.jpg</normaloff>../Multi-Channel-Signal-Viewer-master/up.jpg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="ZoomoutButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Multi-Channel-Signal-Viewer-master/zoom out.jpg</normaloff>../Multi-Channel-Signal-Viewer-master/zoom out.jpg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="ZoominButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Multi-Channel-Signal-Viewer-master/zoom in.jpg</normaloff>../Multi-Channel-Signal-Viewer-master/zoom in.jpg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="DownButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Multi-Channel-Signal-Viewer-master/down.png</normaloff>../Multi-Channel-Signal-Viewer-master/down.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="PuaseButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Multi-Channel-Signal-Viewer-master/pause.jpg</normaloff>../Multi-Channel-Signal-Viewer-master/pause.jpg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Multi-Channel-Signal-Viewer-master/play.jpg</normaloff>../Multi-Channel-Signal-Viewer-master/play.jpg</iconset>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_signal1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hide signal 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_signal2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hide signal 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_signal3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hide signal 3</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comspectropallete">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="minlabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vmin</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxlabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vmax</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>671</width>
      <height>391</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Multi-Channel-Signal-Viewer-master/open.jpg</normaloff>../Multi-Channel-Signal-Viewer-master/open.jpg</iconset>
     </property>
     <addaction name="actionopenSignal_1"/>
     <addaction name="actionopenSignal_2"/>
     <addaction name="actionopenSignal_3"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menucolors">
    <property name="title">
     <string>colors</string>
    </property>
    <addaction name="actioncolorsignal1"/>
    <addaction name="actioncolorsignal2"/>
    <addaction name="actioncolorsignal3"/>
   </widget>
   <widget class="QMenu" name="menuspectro">
    <property name="title">
     <string>spectrogram</string>
    </property>
    <addaction name="menuspectrosignal1"/>
    <addaction name="menuspectrosignal2"/>
    <addaction name="menuspectrosignal3"/>
   </widget>
   <widget class="QMenu" name="menuSpeed">
    <property name="title">
     <string>Speed</string>
    </property>
    <addaction name="actionfast"/>
    <addaction name="actionnormal"/>
    <addaction name="actionslow"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menucolors"/>
   <addaction name="menuspectro"/>
   <addaction name="menuSpeed"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>../Multi-Channel-Signal-Viewer-master/save.png</normaloff>../Multi-Channel-Signal-Viewer-master/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionopenSignal_1">
   <property name="text">
    <string>Signal 1</string>
   </property>
  </action>
  <action name="actionopenSignal_2">
   <property name="text">
    <string>Signal 2</string>
   </property>
  </action>
  <action name="actionopenSignal_3">
   <property name="text">
    <string>Signal 3</string>
   </property>
  </action>
  <action name="actioncolorsignal1">
   <property name="text">
    <string>Signal 1</string>
   </property>
  </action>
  <action name="actioncolorsignal2">
   <property name="text">
    <string>Signal 2</string>
   </property>
  </action>
  <action name="actioncolorsignal3">
   <property name="text">
    <string>Signal 3</string>
   </property>
  </action>
  <action name="menuspectrosignal1">
   <property name="text">
    <string>Signal 1</string>
   </property>
  </action>
  <action name="menuspectrosignal2">
   <property name="text">
    <string>Signal 2</string>
   </property>
  </action>
  <action name="menuspectrosignal3">
   <property name="text">
    <string>Signal 3</string>
   </property>
  </action>
  <action name="actionfast">
   <property name="text">
    <string>fast</string>
   </property>
  </action>
  <action name="actionnormal">
   <property name="text">
    <string>normal</string>
   </property>
  </action>
  <action name="actionslow">
   <property name="text">
    <string>slow</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
